### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by itzharDev**
configure manifest application name  with application class

### CODE REVIEW

File: _includes/android/getting-started.md [hunk 1]
```diff
@@ -64,3 +64,11 @@ public class App extends Application {
   }
 }
 ```
+
+- **Option 3:** Setup `Application` as your application name in `AndroidManifest.xml`
+```xml
+<application ...
+  android:name=".App">
```
Comment (hermanliang) [inline _includes/android/getting-started.md:71]: I think this change can be integrated with `Option 1` on line 23, right?

**Review by hermanliang — CHANGES_REQUESTED** [review]

**Comment by itzharDev** [comment]
I think It's better to put it in the end because u need the class name...

**Comment by hermanliang** [comment]
@itzharDev I know, but it is not an option to set parse. It more likely to be a next step (`Step 3`) of installation.

**Comment by addisonElliott** [comment]
This can be closed since the similar code was merged in #468

File: _includes/android/getting-started.md (hunk 1 above)
Comment (Unic-1) [reply _includes/android/getting-started.md:71]: That would be fine

**Comment by addisonElliott** [comment]
Bump. Can someone close this?

**Pull request closed without merging**